spdx/package.py

```diff
@@ -194,11 +208,12 @@ def validate_mandatory_fields(self, messages):
         return messages
 
     def validate_files(self, messages):
-        if not self.files:
-            messages = messages + ["Package must have at least one file."]
-        else:
-            for f in self.files:
-                messages = f.validate(messages)
+        if self.files_analyzed != False:
+            if not self.files:
+                messages = messages + ["Package must have at least one file."]
```

Comment:
Nit: what about using single quotes there for consistency or using double quotes everywhere?

Reply:
Is this the right line for this comment? I just realized that I forgot to change quotes at lines 135 and 139, it this what you mean?